cartography/client/core/tx.py

```diff
@@ -19,6 +19,17 @@
 logger = logging.getLogger(__name__)
 
 
+def run_write_query(
+    neo4j_session: neo4j.Session, query: str, **parameters: Any
+) -> None:
+    """Execute a write query inside a managed transaction."""
+
+    def _run_query_tx(tx: neo4j.Transaction) -> None:
+        tx.run(query, **parameters).consume()
```

Comment:
I'm not 100% if an explicit consume is necessary anymore. Will double check. Not a blocker though since it's easy to remove in a follow-up.